Pipfile

```diff
@@ -22,7 +22,6 @@ gevent = "==21.12.0"
 gevent-websocket = "==0.10.1"
 taipy-config = {git = "https://git@github.com/avaiga/taipy-config.git@develop"}
 kthread = "==0.2.3"
-werkzeug = "==2.0.3"
```

Comment:
I suppose the right version of werzeug is pulled by flask >= 2.2 ?

Reply:
definitely. the only reason why we need that previously was to run notebook on flask built in server but since we are using gevent for all use case now, this is not needed anymore